test/elements/checkbox-element.js

```diff
@@ -16,56 +16,56 @@ describe('checkbox fields', () => {
     const field = record.form.find('checkbox_field');
 
     field.should.be.instanceof(CheckboxElement);
+    shouldBeNull(field.minLength);
+    shouldBeNull(field.maxLength);
+
+    field.toJSON().should.eql({
```

Comment:
I do wonder if we need all of these fields in the JSON. For dynamic fields, we probably don't need them all. It just bloats the JSON. And that is the only place that toJSON is really used. But generically, I can see an argument to serialize everything.

Maybe just this:
```
type: 'CheckboxField',
      label: 'Checkbox Field',
      key: '92ab',
      data_name: 'checkbox_field',
      description: null,
      default_value: null,
      disabled: false,
      hidden: false,
      required: true,
```

But I am open to other opinions.

Reply:
It's mostly an artifact of inheriting from Element which has all those fields. I'd assume we want this serialization to match what the Fulcrum API would produce as closely as possible.